pkg/activator/handler/handler_test.go

```diff
@@ -268,6 +275,116 @@ func TestActivationHandler(t *testing.T) {
 
 }
 
+// Make sure we return http internal server error when the Breaker is in overflow
+func TestActivationHandler_Overflow(t *testing.T) {
```

Comment:
I suggested below to add a test that different revisions don't "block" each other, when their respective breakers are full. Does it make sense to add such a test here as well or do you think it's enough to test the Throttler itself?

Reply:
I think this one is better - it resembles the tests you suggest.